data/countries_names.txt

```diff
@@ -32474,17 +32474,25 @@
   he = קסטיליה-לה מנצ'ה
   sk = Castile-La Mancha
 
+[Spain_Catalonia]
```

Comment:
How is this string used now?

Reply:
How do you usually manage the closing/resolution of these "conversations" in the pull requests?
I understand that the conversations shall be closed by the launcher of the conversation (in this case, you) once it's been solved or addressed. Or should the one that solves the issue/comment (in this case, me) resolve the conversation?

Reply:
Usually, if you agree with the comment and fix it, then you resolve it. Otherwise, the reviewer may resolve it or leave it open to other people.

We often leave some conversations unresolved to save our limited time )